praw/models/reddit/live.py

```diff
@@ -345,6 +345,24 @@ def discussions(self, **generator_kwargs):
         url = API_PATH['live_discussions'].format(id=self.id)
         return ListingGenerator(self._reddit, url, **generator_kwargs)
 
+    def report(self, type):  # pylint: disable=redefined-builtin
```

Comment:
Cool. Explicit works for me.

I'll give testing an invalid type a try. I just want to ensure no unexpected exceptions occur.